php/libraries/Candidate.class.inc

```diff
@@ -657,17 +657,52 @@ class Candidate
     }
 
     /**
-     * Returns the candidate's calculated age today
+     * Returns the candidate's calculated age today in years, months, days
      *
-     * @return int The candidate's age today
+     * @return array The candidate's age today in an array with values for keys 'year',
+     *               'mon', and 'day'
      */
-    public function getAge(): int
+    public function getAge(): array
     {
-        $ageArray = \Utility::calculateAge(
+        return \Utility::calculateAge(
             $this->candidateInfo["DoB"],
             (new DateTime())->format('Y-m-d')
         );
-        return $ageArray['year'];
+    }
+
+    /**
+     * Returns the candidate's calculated age today in int years
+     *
+     * @return int The candidate's age today as years
+     */
+    public function getAgeYears(): int
+    {
+        return $this->getAge()['year'];
+    }
+
+    /**
+     * Returns the candidate's calculated age today in total number of months
+     *
+     * @return int The candidate's age today in months
+     */
+    public function getAgeInMonths(): int
+    {
+        $age    = $this->getAge();
+        $months = $age['year']*12 + $age['mon'] + ($age['day']/30);
```

Comment:
I think I wouldn't include the days portion for this one. I think rounding up or down could potentially be problematic especially since we're using 1 month = 30 days. The value for "days" doesn't actually have any precision in that case so it's misleading to include it as a significant figure in the calculation.

Either way, all of those caveats should be explained in this function comment as well as in `getAgeInDays()`.